done/static/html/done.html

```diff
@@ -1,10 +1,12 @@
-<div class="done_block">
-  <div class="done_left_spacer"> </div>
-  <div class="done_windshield"> 
-    <div style="display:block; width:100%; height:6px"></div>
-    <div class="done_wipe done_wipe_off"> Undo </div>
-    <div class="done_wiper"> <span class="ui-icon ui-icon-circle-plus ui_icon_white" style="padding-bottom:2px"></span> </div>
-    <div class="done_wipe done_wipe_on"> Mark as complete </div>
+<div class="done_onoffswitch_wrapper">
+  <div class="done_left_spacer"></div>
+  <div class="done_onoffswitch">
+    <input type="checkbox" name="onoffswitch" class="done_onoffswitch-checkbox" id="{id}_myonoffswitch" checked="{done}">  </input>
+    <label class="done_onoffswitch-label" for="{id}_myonoffswitch">
+        <span class="done_aria_hidden">I have completed this</span>
```

Comment:
@pmitros, I recommend changing the class name used here to "sr" to be consistent with platform.  It appears as though you were aiming for your own class naming convention here, but I think that done_aria_hidden could confuse any future developers.  This class has nothing to do with WAI-ARIA, and actually performs the opposite of the ARIA attribute ```aria-hidden``` which actually removes the content from the accessibility tree.  Recommend renaming class to ```sr``` or d```done-offscreen``` which is a better description of what happens to this content.